helpers/content.go

```diff
@@ -84,6 +85,14 @@ func NewContentSpec(cfg config.Provider, logger loggers.Logger, contentFs afero.
 
 	spec.Converters = converterProvider
 	p := converterProvider.Get("markdown")
+	if p == nil {
+		handler := converterProvider.GetMarkupConfig().DefaultMarkdownHandler
```

Comment:
I would prefer if we do this validation here:

https://github.com/gohugoio/hugo/blob/master/markup/markup_config/config.go#L42

Also, Blackfriday isn't deprecated, it's removed, which I think is a "stronger" term.